niworkflows/interfaces/bids.py

```diff
@@ -248,59 +241,77 @@ class _DerivativesDataSinkInputSpec(DynamicTraitedSpec, BaseInterfaceInputSpec):
     check_hdr = traits.Bool(True, usedefault=True, desc='fix headers of NIfTI outputs')
     compress = traits.Bool(desc="force compression (True) or uncompression (False)"
                                 " of the output file (default: same as input)")
-    desc = Str('', usedefault=True, desc='Label for description field')
-    extra_values = traits.List(Str)
-    in_file = InputMultiObject(File(exists=True), mandatory=True,
-                               desc='the object to be saved')
-    keep_dtype = traits.Bool(False, usedefault=True, desc='keep datatype suffix')
     data_dtype = Str(desc='NumPy datatype to coerce NIfTI data to, or `source` to'
                           'match the input file dtype')
+    dismiss_entities = InputMultiObject(
+        traits.Either(None, Str), usedefault=True,
+        desc="a list entities that will not be propagated from the source file")
```

Comment:
Permits setting exceptions to entity propagation (e.g., drop the `echo` of me-epi, which can be useful in some contexts)